astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
```

Comment:
You'll probably laugh at me, but my day job is data analysis and statistics and even though I've learned this in astronomy classes 10 years ago, by now I don't remember what the parallactic angle is.

So I'd be +1 to add a link to wikipedia or something.